xarray/tests/test_conventions.py

```diff
@@ -332,6 +332,17 @@ def test_decode_cf_datetime_transition_to_invalid(self):
 
         assert_array_equal(ds_decoded.time.values, expected)
 
+    @requires_dask
+    def test_decode_cf_with_dask(self):
+        import dask
+        original = Dataset({
+            't': ('t', [0, 1, 2], {'units': 'days since 2000-01-01'}),
+            'foo': ('t', [0, 0, 0], {'coordinates': 'y', 'units': 'bar'}),
+            'y': ('t', [5, 10, -999], {'_FillValue': -999})
+        }).chunk({'t': 1})
+        decoded = conventions.decode_cf(original)
+        assert dask.is_dask_collection(decoded.y.data)
```

Comment:
Is this an appropriate place to test for this? There is no other dask stuff in `test_conventions.py`, but it seemed to make sense there.

Reply:
This looks like a fine place to me.